### ISSUE TITLE
Failed to map the stack to user code

### ISSUE THREAD

**Issue by monkeylumps** [wallabyjs/public#624]
### Issue description or question
I've been trying to setup an example of wallaby working with system js but am having a great deal of trouble.

After lots of playing around I get the tests to run but get the error Failed to map the stack to user code - edit - updated code and new error is testFileIdCanNotBeDetermined - see below
### Wallaby.js configuration file

module.exports = function (wallaby) {
    return {
        files: [
            { pattern: 'node_modules/systemjs/dist/system.js', instrument: false },
            { pattern: 'node_modules/typescript/lib/typescript.js', instrument: false },
            { pattern: 'config.js', instrument: false },
            { pattern: 'scripts/app/*.ts', load: false },
        ],

        tests: [
          { pattern: 'scripts/tests/systemts.spec.ts', load: false }
        ],

        compilers: {
            'scripts/*/*.ts': wallaby.compilers.typeScript({
                "module": 'system', // or amd
                "emitDecoratorMetadata": true,
                "experimentalDecorators": true,
                "noImplicitAny": false
            })
        },

        middleware: (app, express) => {
            app.use('/node_modules',
                    express.static(
                       require('path').join(__dirname, 'node_modules')));
        },

        setup: function (wallaby) {
            wallaby.delayStart();

            var promises = [];
            for (var i = 0, len = wallaby.tests.length; i < len; i++) {
                promises.push(System['import'](wallaby.tests[i].replace(/\.js$/, '')));
            }

            Promise.all(promises).then(function () {
                wallaby.start();
            }).catch(function (e) { setTimeout(function () { throw e; }, 0); });
        },
        debug: true
    };
};
```javascript

test

import {SystemAdder} from '../app/systemts';

describe('test this file', () => {
    it('should add up', () => {
        var thing = new SystemAdder();
        expect(thing.newTotal).toBe(29);
    });
    //    it('should not add up', () => {
    //        var thing = new simpletests.Add();
    //        expect(thing.total).toBe(0);
    //    });
});

it('true is true', () => expect(true).toEqual(true));

ts

systemts.ts

import {Adder} from './imported';

export class SystemAdder {
    newTotal: number;

    constructor() {
        var adder = new Adder();
        this.newTotal = 10 + adder.total;
    }
}
var thing = new SystemAdder();
alert(thing.newTotal);

imported.ts

export class Adder {
    numone: number;
    numtwo: number;
    total: number;

    constructor() {
        this.numone = 7;
        this.numtwo = 12;
        this.total = this.numone + this.numtwo;
    }
}

var thing = new Adder();

alert(thing.total);

```

### Code editor or IDE name and version

Visual Studio 2015

### OS name and version

Windows

Logs

wallaby.js started
core v1.0.240
Wed, 01 Jun 2016 14:40:30 GMT wallaby:project File cache: C:\Users\mothersd\AppData\Local\Temp\.wallaby\projects\mM+IKhycPdm1zMVoorkvfo3Vu08=\cache
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Parallelism for initial run: 6, for regular run: 3
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #0
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #1
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #2
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #3
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #4
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #5
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Web server is listening at 60123
Wed, 01 Jun 2016 14:40:34 GMT wallaby:workers Started run worker instance (immediate) #0
Wed, 01 Jun 2016 14:40:34 GMT wallaby:workers Started run worker instance (immediate) #1
Wed, 01 Jun 2016 14:40:34 GMT wallaby:project File cache requires some updates, waiting required files from IDE
Wed, 01 Jun 2016 14:40:34 GMT wallaby:project File scripts/app/systemts.ts changes are patches only: false
Wed, 01 Jun 2016 14:40:35 GMT wallaby:project Preparing to process scripts/app/systemts.ts
Wed, 01 Jun 2016 14:40:35 GMT wallaby:project Starting processor pool (if not yet started)
Wed, 01 Jun 2016 14:40:35 GMT wallaby:workers Started run worker instance (immediate) #2
Wed, 01 Jun 2016 14:40:36 GMT wallaby:workers Started run worker instance (immediate) #3
Wed, 01 Jun 2016 14:40:36 GMT wallaby:workers Started run worker instance (immediate) #4
Wed, 01 Jun 2016 14:40:37 GMT wallaby:processor No compilers found for scripts/app/systemts.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project File scripts/app/systemts.ts is not instrumented by core
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/app/systemts.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/app/systemts.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project File scripts/app/systemts.ts changes saved, store ts: 636003886848357200, change ts: 1464792034993
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project File scripts/tests/systemts.spec.ts changes are patches only: false
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Preparing to process scripts/tests/systemts.spec.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Starting processor pool (if not yet started)
Wed, 01 Jun 2016 14:40:37 GMT wallaby:processor No compilers found for scripts/tests/systemts.spec.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project File scripts/tests/systemts.spec.ts is not instrumented by core
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/tests/systemts.spec.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/tests/systemts.spec.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project File scripts/tests/systemts.spec.ts changes saved, store ts: 636003888107668700, change ts: 1464792037221
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Stopping process pool
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Running postprocessor
Wed, 01 Jun 2016 14:40:37 GMT wallaby:postprocessor New TypeScript language service is required
Wed, 01 Jun 2016 14:40:37 GMT wallaby:postprocessor Error reading tsconfig file from C:\git repo\jsunittesting\src\jssitetests\tsconfig.json
(Cannot find module 'C:\git repo\jsunittesting\src\jssitetests\tsconfig.json', Cannot read property 'compilerOptions' of undefined)
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Preparing to process scripts/app/imported.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Preparing to process scripts/app/simplets.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Preparing to process scripts/app/systemts.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Preparing to process scripts/tests/systemts.spec.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Processing compiled scripts/app/imported.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Instrumenting scripts/app/imported.js, via process pool: false
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Processing compiled scripts/app/simplets.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Instrumenting scripts/app/simplets.js, via process pool: false
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Processing compiled scripts/app/systemts.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Instrumenting scripts/app/systemts.js, via process pool: false
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Processing compiled scripts/tests/systemts.spec.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Instrumenting scripts/tests/systemts.spec.js, via process pool: false
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Detected single test [true is true] change
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/app/imported.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/app/simplets.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/app/systemts.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/tests/systemts.spec.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/app/imported.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/app/simplets.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/app/systemts.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/tests/systemts.spec.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:project Postprocessor execution finished
Wed, 01 Jun 2016 14:40:38 GMT wallaby:project Test run started; run priority: 3
Wed, 01 Jun 2016 14:40:38 GMT wallaby:project Running all tests
Wed, 01 Jun 2016 14:40:38 GMT wallaby:workers Starting test run, priority: 3
Wed, 01 Jun 2016 14:40:38 GMT wallaby:phantomRunner Starting sandbox [worker #0, session #cr8n6]
Wed, 01 Jun 2016 14:40:38 GMT wallaby:phantomRunner Preparing sandbox [worker #0, session #cr8n6]
Wed, 01 Jun 2016 14:40:38 GMT wallaby:browserRunner Total files to load in sandbox: 3
Wed, 01 Jun 2016 14:40:38 GMT wallaby:browserRunner Sandbox is generated [worker #0, session #cr8n6]: http://localhost:60123/wallaby_sandbox0.html
Wed, 01 Jun 2016 14:40:38 GMT wallaby:phantomRunner Creating page for worker #0
Wed, 01 Jun 2016 14:40:38 GMT wallaby:workers Started run worker instance (immediate) #5
Wed, 01 Jun 2016 14:40:38 GMT wallaby:phantomRunner Phantom page created
Wed, 01 Jun 2016 14:40:38 GMT wallaby:phantomRunner Prepared sandbox [worker #0, session #cr8n6]
Wed, 01 Jun 2016 14:40:38 GMT wallaby:workers Running tests in sandbox [worker #0, session #cr8n6]
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve C:\Users\mothersd\.wallaby\core\tracer.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving C:\Users\mothersd\.wallaby\core\tracer.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /jasmine@2.3.4/framework.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /jasmine@2.3.4/framework.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /jasmine@2.1.3/configurator.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /jasmine@2.1.3/configurator.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /jasmine@2.1.3/reporter.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /jasmine@2.1.3/reporter.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /config.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /config.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /jasmine@2.1.3/starter.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /jasmine@2.1.3/starter.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /scripts/tests/systemts.spec.ts
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /scripts/tests/systemts.spec.ts from cache
Wed, 01 Jun 2016 14:40:39 GMT wallaby:middleware Preparing to serve /scripts/app/systemts.ts
Wed, 01 Jun 2016 14:40:39 GMT wallaby:middleware Serving /scripts/app/systemts.ts from cache
Wed, 01 Jun 2016 14:40:39 GMT wallaby:middleware Preparing to serve /scripts/app/imported.ts
Wed, 01 Jun 2016 14:40:39 GMT wallaby:middleware Serving /scripts/app/imported.ts from disk
Wed, 01 Jun 2016 14:40:39 GMT wallaby:workers Loaded 2 test(s)
Wed, 01 Jun 2016 14:40:39 GMT wallaby:workers Test executed: should add up
Wed, 01 Jun 2016 14:40:39 GMT wallaby:workers Test executed: true is true
Wed, 01 Jun 2016 14:40:39 GMT wallaby:workers Run 2 test(s), skipped 0 test(s)
Wed, 01 Jun 2016 14:40:39 GMT wallaby:workers Sandbox [cr8n6] is responsive, closing it
Finished executing 2 affected test(s)
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Test run finished
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Processed console.log entries
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Processed loading sequences
**Test without test file id, most likely the test did not run { testFileIdCanNotBeDetermined: 'should add up',**
  id: 100001,
  log: undefined,
  filesInvolved: [] }
Test without test file id, most likely the test did not run { testFileIdCanNotBeDetermined: 'true is true',
  id: 100002,
  log: undefined,
  filesInvolved: [] }
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Processed executed tests
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Processed code coverage
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Test run result processed and sent to IDE

**Comment by monkeylumps** [comment]
spotted a bug and updated code accordingly 


new log 

wallaby.js started
core v1.0.240
Wed, 01 Jun 2016 14:40:30 GMT wallaby:project File cache: C:\Users\mothersd\AppData\Local\Temp\.wallaby\projects\mM+IKhycPdm1zMVoorkvfo3Vu08=\cache
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Parallelism for initial run: 6, for regular run: 3
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #0
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #1
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #2
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #3
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #4
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Starting run worker instance #5
Wed, 01 Jun 2016 14:40:31 GMT wallaby:workers Web server is listening at 60123
Wed, 01 Jun 2016 14:40:34 GMT wallaby:workers Started run worker instance (immediate) #0
Wed, 01 Jun 2016 14:40:34 GMT wallaby:workers Started run worker instance (immediate) #1
Wed, 01 Jun 2016 14:40:34 GMT wallaby:project File cache requires some updates, waiting required files from IDE
Wed, 01 Jun 2016 14:40:34 GMT wallaby:project File scripts/app/systemts.ts changes are patches only: false
Wed, 01 Jun 2016 14:40:35 GMT wallaby:project Preparing to process scripts/app/systemts.ts
Wed, 01 Jun 2016 14:40:35 GMT wallaby:project Starting processor pool (if not yet started)
Wed, 01 Jun 2016 14:40:35 GMT wallaby:workers Started run worker instance (immediate) #2
Wed, 01 Jun 2016 14:40:36 GMT wallaby:workers Started run worker instance (immediate) #3
Wed, 01 Jun 2016 14:40:36 GMT wallaby:workers Started run worker instance (immediate) #4
Wed, 01 Jun 2016 14:40:37 GMT wallaby:processor No compilers found for scripts/app/systemts.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project File scripts/app/systemts.ts is not instrumented by core
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/app/systemts.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/app/systemts.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project File scripts/app/systemts.ts changes saved, store ts: 636003886848357200, change ts: 1464792034993
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project File scripts/tests/systemts.spec.ts changes are patches only: false
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Preparing to process scripts/tests/systemts.spec.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Starting processor pool (if not yet started)
Wed, 01 Jun 2016 14:40:37 GMT wallaby:processor No compilers found for scripts/tests/systemts.spec.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project File scripts/tests/systemts.spec.ts is not instrumented by core
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/tests/systemts.spec.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/tests/systemts.spec.ts
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project File scripts/tests/systemts.spec.ts changes saved, store ts: 636003888107668700, change ts: 1464792037221
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Stopping process pool
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Running postprocessor
Wed, 01 Jun 2016 14:40:37 GMT wallaby:postprocessor New TypeScript language service is required
Wed, 01 Jun 2016 14:40:37 GMT wallaby:postprocessor Error reading tsconfig file from C:\git repo\jsunittesting\src\jssitetests\tsconfig.json
(Cannot find module 'C:\git repo\jsunittesting\src\jssitetests\tsconfig.json', Cannot read property 'compilerOptions' of undefined)
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Preparing to process scripts/app/imported.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Preparing to process scripts/app/simplets.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Preparing to process scripts/app/systemts.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Preparing to process scripts/tests/systemts.spec.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Processing compiled scripts/app/imported.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Instrumenting scripts/app/imported.js, via process pool: false
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Processing compiled scripts/app/simplets.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Instrumenting scripts/app/simplets.js, via process pool: false
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Processing compiled scripts/app/systemts.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Instrumenting scripts/app/systemts.js, via process pool: false
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Processing compiled scripts/tests/systemts.spec.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Instrumenting scripts/tests/systemts.spec.js, via process pool: false
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Detected single test [true is true] change
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/app/imported.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/app/simplets.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/app/systemts.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project No preprocessors configured for scripts/tests/systemts.spec.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/app/imported.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/app/simplets.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/app/systemts.js
Wed, 01 Jun 2016 14:40:37 GMT wallaby:project Writing to disk and caching processed file scripts/tests/systemts.spec.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:project Postprocessor execution finished
Wed, 01 Jun 2016 14:40:38 GMT wallaby:project Test run started; run priority: 3
Wed, 01 Jun 2016 14:40:38 GMT wallaby:project Running all tests
Wed, 01 Jun 2016 14:40:38 GMT wallaby:workers Starting test run, priority: 3
Wed, 01 Jun 2016 14:40:38 GMT wallaby:phantomRunner Starting sandbox [worker #0, session #cr8n6]
Wed, 01 Jun 2016 14:40:38 GMT wallaby:phantomRunner Preparing sandbox [worker #0, session #cr8n6]
Wed, 01 Jun 2016 14:40:38 GMT wallaby:browserRunner Total files to load in sandbox: 3
Wed, 01 Jun 2016 14:40:38 GMT wallaby:browserRunner Sandbox is generated [worker #0, session #cr8n6]: http://localhost:60123/wallaby_sandbox0.html
Wed, 01 Jun 2016 14:40:38 GMT wallaby:phantomRunner Creating page for worker #0
Wed, 01 Jun 2016 14:40:38 GMT wallaby:workers Started run worker instance (immediate) #5
Wed, 01 Jun 2016 14:40:38 GMT wallaby:phantomRunner Phantom page created
Wed, 01 Jun 2016 14:40:38 GMT wallaby:phantomRunner Prepared sandbox [worker #0, session #cr8n6]
Wed, 01 Jun 2016 14:40:38 GMT wallaby:workers Running tests in sandbox [worker #0, session #cr8n6]
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve C:\Users\mothersd\.wallaby\core\tracer.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving C:\Users\mothersd\.wallaby\core\tracer.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /jasmine@2.3.4/framework.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /jasmine@2.3.4/framework.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /jasmine@2.1.3/configurator.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /jasmine@2.1.3/configurator.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /jasmine@2.1.3/reporter.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /jasmine@2.1.3/reporter.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /config.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /config.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /jasmine@2.1.3/starter.js
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /jasmine@2.1.3/starter.js from disk
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Preparing to serve /scripts/tests/systemts.spec.ts
Wed, 01 Jun 2016 14:40:38 GMT wallaby:middleware Serving /scripts/tests/systemts.spec.ts from cache
Wed, 01 Jun 2016 14:40:39 GMT wallaby:middleware Preparing to serve /scripts/app/systemts.ts
Wed, 01 Jun 2016 14:40:39 GMT wallaby:middleware Serving /scripts/app/systemts.ts from cache
Wed, 01 Jun 2016 14:40:39 GMT wallaby:middleware Preparing to serve /scripts/app/imported.ts
Wed, 01 Jun 2016 14:40:39 GMT wallaby:middleware Serving /scripts/app/imported.ts from disk
Wed, 01 Jun 2016 14:40:39 GMT wallaby:workers Loaded 2 test(s)
Wed, 01 Jun 2016 14:40:39 GMT wallaby:workers Test executed: should add up
Wed, 01 Jun 2016 14:40:39 GMT wallaby:workers Test executed: true is true
Wed, 01 Jun 2016 14:40:39 GMT wallaby:workers Run 2 test(s), skipped 0 test(s)
Wed, 01 Jun 2016 14:40:39 GMT wallaby:workers Sandbox [cr8n6] is responsive, closing it
Finished executing 2 affected test(s)
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Test run finished
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Processed console.log entries
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Processed loading sequences
Test without test file id, most likely the test did not run { testFileIdCanNotBeDetermined: 'should add up',
  id: 100001,
  log: undefined,
  filesInvolved: [] }
Test without test file id, most likely the test did not run { testFileIdCanNotBeDetermined: 'true is true',
  id: 100002,
  log: undefined,
  filesInvolved: [] }
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Processed executed tests
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Processed code coverage
Wed, 01 Jun 2016 14:40:39 GMT wallaby:project Test run result processed and sent to IDE

**Comment by ArtemGovorov** [comment]
Did you close the issue by mistake or you've sorted it out?

If not, I have created the [repo with your structure](https://github.com/wallabyjs/wallaby-624) and slightly modified config and it works for me. I have removed the `node_modules/typescript/lib/typescript.js` loading as it's not required and added:

```javascript
            System.config({
                defaultJSExtensions: true,
                transpiler: 'none',
                meta: {
                    '*': {
                        scriptLoad: true
                    }
                }
            });
```
to the `setup` function. Wallaby.js compiles TS files and renames them to `.js`, so no runtime TypeScript transpiler is required, and added `scriptLoad: true` for better stack mappings [as described here](https://wallabyjs.com/docs/integration/systemjs.html#module-format).

Let me know if the config works for you, if not, feel free to change the repo I created and send a pull request to reproduce the issue you're having (most likely it'll be just adding your `config.js` file).
